.github/workflows/linux-workflow.yml

```diff
@@ -108,15 +118,40 @@ jobs:
         env:
           PLATFORM: ${{ matrix.platform }}
           COMPILER: ${{ matrix.compiler }}
+        if: ${{ matrix.detail != '-appimage' }}
         run: ./.github/workflows/scripts/linux/install-packages.sh
 
       - name: Generate CMake
         env:
           PLATFORM: ${{ matrix.platform }}
           COMPILER: ${{ matrix.compiler }}
           ADDITIONAL_CMAKE_ARGS: ${{ matrix.cmakeflags }}
+        if: ${{ matrix.detail != '-appimage' }}
         run: ./.github/workflows/scripts/linux/generate-cmake.sh
 
       - name: Build PCSX2
         working-directory: ./build
+        if: ${{ matrix.detail != '-appimage' }}
         run: ../.github/workflows/scripts/linux/compile.sh
+
+      - name: Build AppImage
+        env:
+          PLATFORM: ${{ matrix.platform }}
+          COMPILER: ${{ matrix.compiler }}
+        if: ${{ matrix.detail == '-appimage' }}
+        run: |
+          if [ ${{ matrix.platform }} = x86 ]; then 
+              docker pull quriouspixel/pcsx2:18.04-386
+              docker run -u root --env-file .github/workflows/scripts/linux/docker.env -v $(pwd):/pcsx2 -v "$GITHUB_WORKSPACE/.ccache":/root/.ccache quriouspixel/pcsx2:18.04-386 /bin/bash /pcsx2/.github/workflows/scripts/linux/docker.sh
+          else
+              docker pull quriouspixel/pcsx2:18.04
```

Comment:
i presume these two images are built using the Dockerfiles you included?  It may be worth our time to setup an official pcsx2 dockerhub account so we can update the auto-build and push the images if we have to add/remove packages.

Reply:
Yes, added the files so PCSX2 could build the image,or combine the two as a mulitlib docker.